From 011b50dd5ba94935d5c804cfa0e144230ef3b32e Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 4 Jul 2017 11:23:03 +0800
Subject: [PATCH 2039/5242] MLK-15328-2 arm64: Kconfig: select
 KEYBOARD_SNVS_PWRKEY for i.mx8mq

commit  c1be705563f8882aab1d3099161b5fdd00d81674 from
https://source.codeaurora.org/external/imx/linux-imx.git

Select KEYBOARD_SNVS_PWRKEY for i.MX8MQ by default.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/Kconfig.platforms |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 5e75500..58f52b2 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -325,6 +325,7 @@ config ARCH_FSL_IMX8MQ
 	select PINCTRL_IMX8MQ
 	select PM
 	select SOC_BUS
+	select KEYBOARD_SNVS_PWRKEY
 	help
 	  This enables support for Freescale i.MX8MQ SOC.
 
-- 
1.7.9.5

